### ISSUE TITLE
json creating null values in data.json

### ISSUE THREAD

**Issue by danijeljw-RPC** [repasscloud/lunavpn-lvfucs#13]
When the `data.json` file is created, it is created with `null` values:

```json
{
"peer6_png":null,
"peer6_conf":null,
"peer7_png":null,
"peer7_conf":null,
}
```

**Comment by danijeljw-RPC** [comment]
Error in the code output

**Comment by danijeljw-RPC** [comment]
Fixed earlier than [0b9e7af](https://github.com/repasscloud/lunavpn-lvfucs/commit/0b9e7afc999b71e8e9d4ddad5c5d4a5a9a86a2c8)
